src/sqlfluff/dialects/exasol_keywords.py

```diff
@@ -871,3 +878,53 @@
     "CONNECT_BY_ROOT",
     "CONNECT_BY_ISCYCLE",
 ]
+
+SYSTEM_PARAMETERS = [
+    "HASHTYPE_FORMAT",
+    "TIME_ZONE",
+    "TIME_ZONE_BEHAVIOR",
+    "TIMESTAMP_ARITHMETIC_BEHAVIOR",
+    "NLS_DATE_FORMAT",
+    "NLS_TIMESTAMP_FORMAT",
+    "NLS_DATE_LANGUAGE",
+    "NLS_FIRST_DAY_OF_WEEK",
+    "NLS_NUMERIC_CHARACTERS",
+    "DEFAULT_LIKE_ESCAPE_CHARACTER",
+    "QUERY_CACHE",
+    "QUERY_TIMEOUT",
+    "CONSTRAINT_STATE_DEFAULT",
+    "PROFILE",
+    "SCRIPT_LANGUAGES",
+    "SQL_PREPROCESSOR_SCRIPT",
+    "PASSWORD_SECURITY_POLICY",
+    "PASSWORD_EXPIRY_POLICY",
+    "DEFAULT_CONSUMER_GROUP",
+    "SCRIPT_OUTPUT_ADDRESS",
+    "TEMP_DB_RAM_LIMIT",
+    "USER_TEMP_DB_RAM_LIMIT",
+    "SESSION_TEMP_DB_RAM_LIMIT",
+    "ST_MAX_DECIMAL_DIGITS",
+]
+
+SESSION_PARAMETERS = [
+    "HASHTYPE_FORMAT",
+    "TIME_ZONE",
+    "TIME_ZONE_BEHAVIOR",
+    "TIMESTAMP_ARITHMETIC_BEHAVIOR",
+    "NLS_DATE_FORMAT",
+    "NLS_TIMESTAMP_FORMAT",
+    "NLS_DATE_LANGUAGE",
+    "NLS_FIRST_DAY_OF_WEEK",
+    "NLS_NUMERIC_CHARACTERS",
+    "DEFAULT_LIKE_ESCAPE_CHARACTER",
+    "QUERY_CACHE",
+    "QUERY_TIMEOUT",
+    "CONSTRAINT_STATE_DEFAULT",
+    "PROFILE",
+    "SCRIPT_LANGUAGES",
+    "SQL_PREPROCESSOR_SCRIPT",
+    "NICE",
+    "SCRIPT_OUTPUT_ADDRESS",
+    "SESSION_TEMP_DB_RAM_LIMIT",
+    "ST_MAX_DECIMAL_DIGITS",
```

Comment:
And here.

BTW note a lot of overlap with the list above but guess it's different enough to be two lists and not really worth while having a common list? Probably not but thought I'd ask.